examples/nodejs/cache/doc-example-files/doc-examples-js-apis.ts

```diff
@@ -1455,6 +1457,10 @@ async function main() {
   await example_API_LeaderboardLength(leaderboard);
   await example_API_LeaderboardRemoveElements(leaderboard);
   await example_API_LeaderboardDelete(leaderboard);
+  await example_API_LeaderboardFetchByRankPagination(leaderboard);
```

Comment:
@anitarua I noticed that none of these paginated ones were getting called, idk if that was intentional? If so I can revert this part.

Reply:
whoops, definitely didn't mean to leave those out, thanks for catching that!